runtime/arm64.S

```diff
@@ -270,18 +270,19 @@ FUNCTION(caml_alloc3)
 
 FUNCTION(caml_allocN)
         CFI_STARTPROC
-        sub     ALLOC_PTR, ALLOC_PTR, ARG
+        sub     ALLOC_PTR, ALLOC_PTR, ADDITIONAL_ARG
         cmp     ALLOC_PTR, ALLOC_LIMIT
         b.lo    L(caml_call_gc)
         ret
         CFI_ENDPROC
         END_FUNCTION(caml_allocN)
 
 /* Call a C function from OCaml */
-/* Function to call is in ARG */
+/* Function to call is in ADDITIONAL_ARG */
 
 FUNCTION(caml_c_call)
         CFI_STARTPROC
+    /* I think using x19 here should be a bug */
```

Comment:
What?

Reply:
I have no idea either, probably I thought that x19 was volatile, will remove it thanks